docs/advanced_reference/commands/autonomy_build-image.md

```diff
@@ -1,109 +1,86 @@
+This command builds the Docker runtime images for the agent defined in a service configuration file `service.yaml`. By default, the command tags the generated images as
 
-To build an image for a service navigate to the service directory, and run
-
-```bash
-autonomy build-image
+```
+<author>/oar-<agent_package>:<package_hash>
 ```
 
-Or if you want to build an image for a specific agent package without fetching a service, run
+where `<author>` is the author name from the local CLI config (specified with `autonomy init`), `oar` stands for "Open Autonomy Runtime", and `<agent_package>:<package_hash>` is the `PUBLIC_ID` of the agent. These default tags can be modified using certain options described below.
 
-```bash
-autonomy build-image PUBLIC_ID
-```
+!!! warning "Important"
 
-where `PUBLIC_ID` refers to the agent public ID (with hash included).
+    The images are built by fetching the packages from the IFPS registry. Therefore, before running the command, make sure that all the required packages are published on the IPFS registry.
 
 ## Usage
 
 ```bash
-autonomy build-image [OPTIONS] [PUBLIC_ID_OR_HASH]
+autonomy build-image [OPTIONS] [PUBLIC_ID]
 ```
 
 ## Options
-```
---service-dir PATH
-```
-:   Path to service directory.
 
-```
---version TEXT
-```
-:   Specify tag version for the image.
+`--service-dir PATH`
+:   Path to the service directory.
 
-```
---image-author TEXT  
-```
-:   Specify author name for docker image.
+`--version TEXT`
+:   Tag version for the image.
 
-```
---dev
-```
-:   Build development image.
+`--image-author TEXT`
+:   Author name for the image.
 
-```
---pull
-```
-:   Pull latest dependencies.
+`--dev`
+:   Build the agent image in [`dev` mode](../developer_tooling/dev_mode.md).
 
-```
---help
-```
-:   Show the help message and exit.
+`--pull`
+:   Pull the latest dependencies when building the image.
 
+`--help`
+:   Show the help message and exit.
 
 ## Examples
 
-### To build a runtime image, run
+* Build the agent image for a service located in the folder `your_service`, which should contain the service configuration file `service.yaml`:
 
-```bash
-$ cd /my_service
-$ autonomy build-image
-```
+    ```bash
+    autonomy build-image --service-dir /your_service
+    ```
 
-or
+    Or, alternatively:
 
-```bash
-$ autonomy build-image --service-dir /my_service
-```
+    ```bash
+    cd /your_service
+    autonomy build-image
+    ```
 
-Builds the agent images for the service located in the directory `my_service`. 
+* Build the agent image for a specific service without fetching it:
 
-### To build a runtime image using a public ID
+    ```bash
+    autonomy build-image <agent_package>:<package_hash>
+    ```
 
-```bash
-$ autonomy build-image <author>/<agent_package>:<package_hash>
-```
+    where `<agent_package>:<package_hash>` is the `PUBLIC_ID` of the agent.
```

Comment:
The public ID follows `<author>/<package_name>:<package_hash>` format

Reply:
fixed